docs/database/gravity-groups.md

```diff
@@ -0,0 +1,35 @@
+## Group management
+Any domain on the individual lists as well as the adlists entries can be managed through groups. This allows not only grouping them together to highlight their relationship, but also enabling/disabling them together if one, for instance, wants to visit a specific service only temporarily.
```

Comment:
I wonder if someone could confuse this to mean that any blocklist domain can also be on a group due to:
> Any domain on the individual lists as well as the adlists entries can be managed through groups.

Perhaps it's worthwhile to clarify this.

Reply:
@DL6ER this is the only remaining review item.